<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color:#F0EEEE ;

border-top-left-radius: 6px 6px;
border-top-right-radius: 6px 6px;
border-bottom-right-radius: 6px 6px;
border-bottom-left-radius: 6px 6px;

}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lbTitle">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
color: #252424;
font: bold 16px;
font-family: Ubuntu, Helvetica, sans-serif;
}</string>
        </property>
        <property name="text">
         <string>&lt;List name&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">QLabel{
color:rgb(85, 85, 85);
font:bold;
font-family: Ubuntu, Helvetica, sans-serif;
}</string>
          </property>
          <property name="text">
           <string>Search</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLineEdit" name="leSearch">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>96</width>
              <height>0</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit { 
border: 1px solid #999999; 
padding: 2px 2px 2px 2px;
min-width: 90px;
height: 20px;

border-top-left-radius: 3px ;
border-top-right-radius: 0px solid white;
border-bottom-right-radius: 0px ;
border-bottom-left-radius: 3px;

}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbLupita">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>30</width>
              <height>26</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel { 
background: white;
border: 1px solid #999999; 
padding: 2px 2px 2px 2px;

height: 30px;
width:26px;

border-top-left-radius: 0px ;
border-top-right-radius: 3px;
border-bottom-right-radius: 3px;
border-bottom-left-radius: 0px;

border-left: 0px solid white;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="images_rc/images.qrc">:/newPrefix/find.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">QLabel{
color:rgb(85, 85, 85);
font:bold;
font-family: Ubuntu, Helvetica, sans-serif;
}</string>
          </property>
          <property name="text">
           <string>Filters</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbFilters">
          <property name="styleSheet">
           <string notr="true">QComboBox {
border: 1px solid #999999; 
border-radius: 4px;
padding: 2px 2px 2px 2px;
min-width: 6em;
}

QComboBox::down-arrow {
image: url(:newPrefix/arrow_down.png);
height: 10px;
}
QComboBox:editable {
background: white;
border-radius: 0px;
}
QComboBox::drop-down { border-style: none}</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="btNew">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>90</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu,Helvetica,sans-serif</family>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>New</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btEdit">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Edit</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btDelete">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="lbItemsCount">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>42</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
color: rgb(147, 147, 147);
font: 10px;
font-family: Ubuntu, Helvetica, sans-serif;
}</string>
        </property>
        <property name="text">
         <string>0 items(s) listed</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="twItems">
      <property name="font">
       <font>
        <family>Ubuntu,Helvetica,sans-serif</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QHeaderView::section {
border-radius: 3px;
padding: 2px 4px 2px 4px;

/* Fuente */
color: #3A3939;
text-align: left;
font-family: Ubuntu, Helvetica, sans-serif;

background-color:rgb(244, 244, 244);
/*qlineargradient(spread:reflect, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(240, 240, 240, 255), stop:0.509174 rgba(250, 250, 250, 255), stop:0.963303 rgba(255, 255, 255, 255));*/
/* Bordes */
border-bottom: 1px solid #E3E3E3;
border-top: 1px solid #FFFFFF;
padding:6px;
border-top-left-radius: 0px;
}

QTableWidget{
font-family: Ubuntu, Helvetica, sans-serif;
/*font-size: 12px;*/
text-align: left;
border: 1px solid #BFBFBF;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::AnyKeyPressed</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="horizontalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::NoPen</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>24</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images_rc/images.qrc"/>
 </resources>
 <connections/>
</ui>
